proto/nodeinfo_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 The ThunderDB Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package proto
+
+import (
+	"testing"
+
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/thunderdb/ThunderDB/crypto/hash"
+)
+
+func TestNode_InitNodeCryptoInfo(t *testing.T) {
+	NewNodeIDDifficultyTimeout = 60000 * time.Millisecond
+	Convey("InitNodeCryptoInfo", t, func() {
+		node := NewNode()
+		err := node.InitNodeCryptoInfo()
+		hash_tmp := hash.DoubleHashH(append(node.PublicKey.SerializeCompressed(),
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
don't use underscores in Go names; var hash_tmp should be hashTmp